clients/filesystem-hadoop3/src/test/java/org/apache/gravitino/filesystem/hadoop/TestGvfsBase.java

```
@@ -290,6 +290,12 @@ public void testRequestHeaders()
                 .withBody(getJsonString(new VersionResponse(Version.getCurrentVersionDTO()))));
 
     try (FileSystem fs = new Path("gvfs://fileset/").getFileSystem(configuration)) {
+      // Trigger lazy initialization by accessing a path
+      try {
+        fs.exists(new Path("gvfs://fileset/catalog/schema/fileset/file.txt"));
+      } catch (Exception e) {
+        // Ignore errors, we just want to trigger client initialization
```

Should we throw the exception?